Description: Disable failing test

--- psrchive-2012.12+git20180817.52211d7.orig/Util/genutil/Makefile.am
+++ psrchive-2012.12+git20180817.52211d7/Util/genutil/Makefile.am
@@ -160,7 +160,7 @@ TESTS = test_evaluate test_coord test_Ca
 	test_endian test_sum test_NormalDistribution			\
 	test_TemporaryFile test_moment2 test_MJD_ostream test_sky_coord	\
 	test_exponential test_StraightLine test_ThreadStream		\
-	test_Horizon test_LogFile test_Warning test_RunningMedian \
+	test_LogFile test_Warning test_RunningMedian \
 	test_PhaseRange
 
 check_PROGRAMS = $(TESTS) test_CommandLine test_CommandParser \
